extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -162,10 +166,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
+        this.element.parentElement);
     return this.ensureInitialized_();
   }
 
   /** @override */
+  unlayoutCallback() {
+    const ampDoc = this.getAmpDoc();
+    Transport.doneUsingCrossDomainIframe(ampDoc.win.document,
+        this.config_['transport']);
+    ResponseMap.remove(ampDoc, this.config_['transport']['type']);
```

Comment:
What if config has no transport?  Or no type?  Should this only run for amp analytics elements that utilize 3p frame?  Consider wrapping in if (this.config_['transport'] && this.config_['transport']['iframe']) {